test/units/TEST-74-AUX-UTILS.ssh.sh

```diff
@@ -43,8 +43,13 @@ usermod -U root
 
 mkdir -p /etc/ssh
 test -f /etc/ssh/ssh_host_rsa_key || ssh-keygen -t rsa -C '' -N '' -f /etc/ssh/ssh_host_rsa_key
-echo "PermitRootLogin yes" >> /etc/ssh/sshd_config
-echo "LogLevel DEBUG3" >> /etc/ssh/sshd_config
+
+# Use /etc/ssh/sshd_config.d/ to not overwrite sshd config when config is in
+# /usr/etc/ssh/sshd_config
+{
+    echo "PermitRootLogin yes"
+    echo "LogLevel DEBUG3"
+} >/etc/ssh/sshd_config.d/test.conf
```

Comment:
Please do `mkdir -p /etc/ssh/sshd_config.d/` before writing, for stability.

Also, please drop test.conf on exit, by using `trap`.

Reply:
Thank you for the review ! `mkdir` and `trap` used in the last changes